lib/chef/http/authenticator.rb

```diff
@@ -104,6 +152,51 @@ def load_signing_key(key_file, raw_key = nil)
         raise Chef::Exceptions::InvalidPrivateKey, msg
       end
 
+      def self.retrieve_certificate_key(client_name)
+        require "openssl" unless defined?(OpenSSL)
+
+        if ChefUtils.windows?
```

Comment:
instead of nesting this , we could open with `return false unless ChefUtils.windows?` at the top of the function

Reply:
I remember now, I put that block in there like that because I was setting this up to run on both Linux and Mac down the road.